src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -57,7 +57,7 @@
             TextTrimming="CharacterEllipsis"
             Margin="0,-3,5,0"
             Content="{Binding Title}"
-            ToolTip="{Binding Title}"
+            ToolTip="View Pull Request details"
```

Comment:
Each project has its own resx file, I forget which project this code belongs to 😆 but it should be there.